cmd/werf/common/storage.go

```diff
@@ -0,0 +1,80 @@
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/werf/werf/v2/pkg/container_backend"
+	"github.com/werf/werf/v2/pkg/storage/manager"
+)
+
+type Option func(*NewStorageManagerConfig)
```

Comment:
We need a separate package or unique name.

Reply:
resolved